### PR TITLE
C string formatting cheat sheet: Added two more specifiers

### PR DESCRIPTION
**Pull request by mayank7**
Added two more specifiers to c string formatting cheat sheet.

IA Page: https://duck.co/ia/view/c_string_formatting_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
## [C String Formatting Cheat Sheet](https://duck.co/ia/view/c_string_formatting_cheat_sheet)

**Description**: Provides a cheat sheet for printf string formatting in the C standard library.

**Example Query**: [C String Formatting Cheat Sheet](https://beta.duckduckgo.com/?q=C String Formatting Cheat Sheet), [](https://beta.duckduckgo.com/?q=)

**Tab Name**: Answer

**Source**:  

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/c_string_formatting_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/c_string_formatting_cheat_sheet)*

File: share/goodie/cheat_sheets/json/c-string-formatting.json [hunk 1]
```diff
@@ -67,6 +67,14 @@
         "val": "Use shortest representation, %E or %F"
         },
         {
+        "key": "hi",
+        "val": "Signed Integer(Short)"
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/c-string-formatting.json]: Do you have a reference for these that we can add in?

Comment (mayank7) [inline share/goodie/cheat_sheets/json/c-string-formatting.json]: @mintsoft I found it here
http://www.codeforwin.in/2015/05/list-of-all-format-specifiers-in-c.html

Comment (mintsoft) [inline share/goodie/cheat_sheets/json/c-string-formatting.json]: That's cool; I've just verified that with a secondary source as they're not tokens that I recognise off the bat as it were

**Pull request merged**